astropy/utils/console.py

```diff
@@ -365,7 +365,7 @@ def strip_ansi_codes(s):
     """
     Remove ANSI color codes from the string.
     """
-    return re.sub('\033\[([0-9]+)(;[0-9]+)*m', '', s)
+    return re.sub(r'\033\[([0-9]+)(;[0-9]+)*m', '', s)
```

Comment:
Is this in the new python?
Anyway, might as well try to have it make most sense. Certainly:
```
re.search("\003", r"mister\003smith")
```
does not return any matches

It does seem a bit troublesome that
```
re.search(r"\003", r"mister\003smith")
```
also does not return any matches... While
```
re.search(r"\\003", r"mister\003smith")
Out[7]: <_sre.SRE_Match object; span=(6, 10), match='\\003'>
```